### PR TITLE
tgchat sorting fix, adds "Mentor PMs" tab

### PR DESCRIPTION
**Pull request by imsynthex**
## About The Pull Request

Fixes some minor things to do with tgchat sorting and what not.

## Why It's Good For The Game

Mentor PMs and some radio channels used to be unsorted. Makes it way easier to have a tab with just Mentor PMs or radio channels.

## Changelog
:cl:
add: adds a "Mentor PM" tab to tgchat settings, adds most antag comms to "Radio" tab
fix: fixes "Radio" tab not showing supply, service, centcom and AI private channels
/:cl:

### CODE REVIEW

File: tgui/packages/tgui-panel/chat/constants.js [hunk 1]
```diff
@@ -58,13 +59,13 @@ export const MESSAGE_TYPES = [
     type: MESSAGE_TYPE_RADIO,
     name: 'Radio',
     description: 'All departments of radio messages',
```
Comment (francinum) [inline tgui/packages/tgui-panel/chat/constants.js]:
```suggestion
    name: 'Radio',
    description: 'All departments of radio messages, along with various antagonist private channels',
```

**Comment by PowerfulBacon** [comment]
This adds the tab, but does this actually put the mentor messages in that tab?

**Comment by imsynthex** [comment]
> This adds the tab, but does this actually put the mentor messages in that tab?

Yes, the tab has Mentor PMs

`selector: '.mentorfrom, .mentorto, .mentorhelp',`

**Comment by imsynthex** [comment]
That works, for now

**Comment by imsynthex** [comment]
Gotta PR this again smh 😔

**Pull request closed without merging**